Resources/Prototypes/_Moffstation/PersonalItems/Items/privatemumbles/Squeaks/squeakshampter.yml

```diff
@@ -22,3 +24,4 @@
   - !type:PersonalItemLoadoutEffect
     character:
     - Squeaks
+    - Squeaks!
```

Comment:
Oh also I did this because I remembered that privatemumbles "spells" it this way